guides/_subsections/create-a-custom-target.md

```diff
@@ -9,13 +14,15 @@ By default, Cryostat automatically discovers target JVMs over remote Java Manage
   <li>
     {% capture open-custom-target-form-text %}
     <p>
-      Search for <i>Custom Target</i> tab and click <i>Create</i> to open the <i>Custom Target</i> form.
+      Search for <i>Custom Target</i> tab and click <i>Create</i> to open the
+      <i>Custom Target</i> form.
       <figure>
         <a href="{{ site.url }}/images/2.3.0/custom-target-1.png" target="_blank">
           <img src="{{ site.url }}/images/2.3.0/custom-target-1.png">
         </a>
         <figcaption>
-          Use the <i>Custom Target</i> form to tell Cryostat about a JVM application that are not automatically discovered.
+          Use the <i>Custom Target</i> form to tell Cryostat about a JVM
+          application that are not automatically discovered.
```

Comment:
"... about JVM applications that are not automatically discovered." or "... about a JVM application that is/was not automatically discovered."